x-pack/osquerybeat/scripts/mage/package.go

```diff
@@ -15,8 +16,16 @@ func CustomizePackaging() {
 	for _, args := range devtools.Packages {
 		distFile := distro.OsquerydDistroPlatformFilename(args.OS)
 
+		// The minimal change to fix the issue for 7.13
+		// https://github.com/elastic/security-team/issues/1162
```

Comment:
Link to private issue?

Reply:
discussed over slack: leaving for now to get CI build done